js/src/launchAsync.ts

```diff
@@ -179,11 +179,9 @@ export function close(): void {
 
 // The subdomain must be alphanumeric, and may contain '-',
 // as long as the '-' does not start or end the subdomain.
-// The subdomain can also be falsy (null/undefined/'') in order to support our legacy token format,
-// though in the future the legacy token will not be supported.
 export function isValidSubdomain(subdomain: string): boolean {
     if (!subdomain) {
-        return true;
+        return false;
```

Comment:
this will require a change in https://github.com/microsoft/immersive-reader-sdk/blob/master/js/tests/launchAsync.test.ts
for the isValidSubdomain tests